Tests/MeshLib/TestRasterDataToMesh.cpp

```diff
@@ -0,0 +1,82 @@
+/**
+ * \copyright
```

Comment:
```suggestion
 * \file
 * \copyright
```
Otherwise doxygen renders pages wrong. #2660

Reply:
✔️